roles/zabbix_server/tasks/Debian.yml

```diff
@@ -129,116 +95,19 @@
   tags:
     - install
 
-# On certain 18.04 images, such as docker or lxc, dpkg is configured not to
+# On certain Ubuntu container images, dpkg is configured not to
 # install files into paths /usr/share/doc/*
-# Since this is where Zabbix installs its database schemas, we need to allow
-# files to be installed to /usr/share/doc/zabbix*
-- name: "Debian | Check for the dpkg exclude line"
-  ansible.builtin.command: grep -F 'path-exclude=/usr/share/doc/*' /etc/dpkg/dpkg.cfg.d/excludes
-  register: dpkg_exclude_line
-  failed_when: false
-  changed_when: false
-  check_mode: false
-  become: true
-  tags:
-    - install
-
-- name: "Debian | Allow Zabbix dpkg installs to /usr/share/doc/zabbix*"
+# Since this is where Zabbix 5.0 installs its database schemas, we need to allow
+# files to be installed to /usr/share/doc/zabbix-server-*
+- name: "Debian | Allow Zabbix dpkg installs to /usr/share/doc/zabbix-server-*"
   ansible.builtin.lineinfile:
     path: /etc/dpkg/dpkg.cfg.d/excludes
-    line: "path-include=/usr/share/doc/zabbix*"
+    create: true
+    line: 'path-include=/usr/share/doc/zabbix\-server\-*'
```

Comment:
Yeah, I discovered a similar task in molecule/zabbix_proxy/prepare.yml, and I think I questioned (in this commit) whether or not this should be done there instead. I've since moved it there. But yeah, it should get removed eventually.

Reply:
Done in 3923029571441fa265358fe1384617a2bb70be08